lib/patch_ruby.rb

```diff
@@ -35,6 +35,7 @@
 require 'patch_ruby/models/order'
 require 'patch_ruby/models/order_list_response'
 require 'patch_ruby/models/order_response'
+require 'patch_ruby/models/parent_technology_type'
```

Comment:
@kleinjm to confirm, this got added automatically, right? (we used to have to do this manually, but hopefully our latest changes fixed this)

Reply:
100% automatic 😎